samples/storageclass/storageclass-xfs.yaml

```diff
@@ -31,10 +31,10 @@ parameters:
   # Uncomment the line below if you want to use mkfsFormatOption
   # mkfsFormatOption: "<mkfs_format_option>" # Insert file system format option
   # Filesytem type for volumes created by storageclass
-  # Allowed values: "ext4" or "xfs"
-  # Optional: true
-  # default: "ext4"
-  FsType: xfs
+  # Default value: None if defaultFsType is not mentioned in values.yaml
+  # Else defaultFsType value mentioned in values.yaml
+  # will be used as default value
+  csi.storage.k8s.io/fstype: xfs
```

Comment:
Just wondering if we need to call out what users will get if they have None in values.yaml in here.

Reply:
if they mention None here K8s will not set any default value unlike before it used to set ext4, so the installation will fail off

Reply:
@jooseppi-luna @JacobGros  - Can you confirm the above? Per our discussion, my assumption was that None would default to either xfs or ext4 based on the storage class used, but I could be mistaken. 
@shanmydell  - why provide None as an option when it we know it will fail?

Reply:
no @bharathsreekanth , there has been a PR in k8s and the behavior has been changed, we have to mention the Default value if user doesn't provide one

Reply:
Thanks @shanmydell  - Couldnt the default value be one of ext4 or xfs?

Reply:
https://github.com/kubernetes-csi/external-provisioner/pull/400/files @bharathsreekanth According to the PR it has been changed

Reply:
That helps, thanks.